Shivani_Bansal/calculator/Calculator.js

```diff
@@ -0,0 +1,102 @@
+function insert(num){
+    // calculator.display.value += num
+    console.log(num,num.target, "ab")
+
+    document.getElementById("display").value = document.getElementById("display").value + num;
+    
+}
+
+function solve(){
+    txt = document.getElementById("display").value;
+    l = txt.length;
```

Comment:
Never use short/truncated names for variables, Always use meaningful name for functions and varaibles
l (poor readability) --> textLength (improved readability)